src/ccl_halofit.c

```diff
@@ -24,7 +24,7 @@ void compute_chi(double a, ccl_cosmology *cosmo, double * chi, int * stat);
 
 static double zdrag_eh(ccl_parameters *params) {
   // eqn 4 of Eisenstein & Hu 1998
-  double OMh2 = params->Omega_m * params->h * params->h;
+  double OMh2 = (params->Omega_c + params->Omega_b) * params->h * params->h;
```

Comment:
zdrag only enters in the EH calculation?

Reply:
yep this function is local to the halofit code.